Architecture_Engine/Create/Theatron/TierProfile.cs

```diff
@@ -55,26 +67,49 @@ public static TierProfile TierProfile(ProfileParameters parameters, Point lastPo
             return tierProfile;
         }
 
-
-        /***************************************************/
         [Description("Scale, rotate and translate a TierProfile")]
         [Input("originalSection", "TierProfile to transform")]
         [Input("scale", "Scaling amount")]
         [Input("source", "Origin point for the transform")]
         [Input("target", "Target point for the transform")]
         [Input("angle", "Rotation angle")]
-        //this should be a modify method
+        [PreviousVersion("4.2", "BH.Engine.Architecture.Theatron.Create.TransformProfile(BH.oM.Architecture.Theatron.TierProfile, BH.oM.Geometry.Vector, BH.oM.Geometry.Point, BH.oM.Geometry.Point, System.Double)")]
```

Comment:
this is not needed any more, I would revert back to the comment about the method to be moved to `Modify`